conformance/eexp/basic_system_macros.ion

```diff
@@ -0,0 +1,91 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+// void
+
+(ion_1_1
+  (each (toplevel 1 ('#$:$ion:void') 2)
+        (toplevel 1 ('#$:$ion:0')    2)
+        (toplevel 1 ('#$:void')      2)
+        (produces 1                  2)))
+
+// TODO should this stay true?
+(ion_1_1
+  (toplevel 1 ('#$:$ion:void' true ('#$:void' false)) 2)
+  (produces 1 2))
+
+
+// values
+
+(ion_1_1
+  (each (toplevel 1 ('#$:$ion:values') 2)
```

Comment:
Just curious—do you have something that is formatting this for you, or are you formatting all of these by hand?

Reply:
The Ion IntelliJ plugin helps a bit, but it also mangles indentation so a fair amount of hand indentation is necessary.

Reply:
We have this to fix the indentation: https://github.com/amazon-ion/ion-intellij-plugin/pull/65
But I think we got hung up on whether to use lisp or list style for s-expressions, or if we need to make it configurable. I'm assuming you prefer the style that you're using in these tests?

Reply:
I commented on that issue, thanks for pointing it out.  S-expressions shouldn't be formatted like lists; they exist to represent code, and list-like formatting makes code unreadable.

I also cut a new issue about the indentation mangling.